test/autocomplete.test.js

```diff
@@ -1,31 +1,25 @@
-import '../autocomplete.js';
 import '../autocomplete-input-text.js';
-import '@polymer/iron-test-helpers/mock-interactions.js';
-import { expect, fixture, html, oneEvent } from '@brightspace-ui/testing';
-import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
-import { flush as flush$0 } from '@polymer/polymer/lib/utils/flush.js';
+import { expect, fixture, html, oneEvent, sendKeys, waitUntil } from '@brightspace-ui/testing';
 
 const basicFixture = html`
-		<d2l-labs-autocomplete id="basic-test">
-			<input slot="input">
-		</d2l-labs-autocomplete>
+		<d2l-labs-autocomplete-input-text id="basic-test">
+		</d2l-labs-autocomplete-input-text>
```

Comment:
Don't need `id`s on these I don't think.
```suggestion
		<d2l-labs-autocomplete-input-text>
		</d2l-labs-autocomplete-input-text>
```